libr/core/cmd_write.c

```diff
@@ -1555,7 +1555,7 @@ static int cmd_write(void *data, const char *input) {
 							RAsmCode *ac = r_asm_massemble (core->assembler, b);
 							char* hex = r_asm_code_get_hex (ac);
```

Comment:
Use ac->bytes and ac->len, but this should be rewritten to use the RBuffer API instead of having two separate fields

Reply:
and use `r_io_write_at (core->io, addr, ac->code, ac->len);`